cases/performance/file_to_blackhole/vector.toml

```diff
@@ -0,0 +1,19 @@
+data_dir = "data"
+
+[sources.internal]
+type = "internal_metrics"
+
+[sources.file]
+type = "file"
+max_line_bytes = 102400
```

Comment:
Unless I'm missing something, I think this should explain why we see a drop-off right at 100kb line size 😄 . If we add another zero it should align better with the generated data.

That being said, I'm not terribly worried about extremely large individual lines. My assumption is that 100kb is a reasonable upper limit for "normal" use cases.

Reply:
D'oh, good eye. I thought I had that set to 1MB.

Reply:
Well, that's good, correcting the line size actually makes the results I'm seeing more explicable. I'll add a section to the notes tomorrow. 

I can for sure produce a situation where vector falls behind and never catches up and this happens cleanly _without_ my `max_line_bytes` bug muddying the waters. If I have `file_gen` 10 files at 500Kb line bytes and adjust `max_line_bytes` to _actually_ be 1MB the effect still exists, except now the throughput from file source is 5MB where I'd expect 25MB. Likewise if `file_gen` produces 255 files at 100Kb line bytes the throughput from the file source is ~430Mb/s where `file_gen` is producing at ~630Mb/s.

Reply:
If I adjust `file_gen` to produce 255 10Kb line files the numbers look like 600Mb/s from `file_gen` and 400Mb/s from the file source. Excellent result. I'll produce a clearer table tomorrow, but these numbers are promising.